features/roomdetails/impl/src/main/kotlin/io/element/android/features/roomdetails/impl/members/RoomMemberListView.kt

```diff
@@ -80,16 +94,41 @@ fun RoomMemberListView(
                 .padding(padding),
             verticalArrangement = Arrangement.spacedBy(16.dp),
         ) {
-            UserListView(
-                state = state.userListState,
-                onUserSelected = ::onUserSelected,
-            )
+            Column {
+                RoomMemberSearchBar(
+                    modifier = Modifier.fillMaxWidth(),
+                    query = state.searchQuery,
+                    state = state.searchResults,
+                    active = state.isSearchActive,
+                    onActiveChanged = { state.eventSink(RoomMemberListEvents.OnSearchActiveChanged(it)) },
+                    onTextChanged = { state.eventSink(RoomMemberListEvents.UpdateSearchQuery(it)) },
+                    onUserSelected = { onUserSelected(it) }
```

Comment:
Apparently, using function references works better in Compose. I think we could just use `::onUserSelected` here instead of the lambda.

Reply:
Sure, can you explain what "better" means in this context?

Reply:
Function references are marked as `@Stable` and won't trigger recompositions, while lambda can. It's just a bit of optimization.